README.md

```diff
@@ -139,6 +172,59 @@ sent to a server implementing the `Greeter` service with:
       ...))
 ```
 
+If the service implements client-side streaming `message` should be a list
+of `hello-request` messages to be sent to the server. If the service implements
+server-side streaming then response will contain a list of `hello-reply`
+messages.
+
+#### Asynchronous Client Streaming
+
+For streaming calls we create:
+
+-   `cl-protobufs.testing-rpc:*/start`
+-   `cl-protobufs.testing-rpc:*/send`
+-   `cl-protobufs.testing-rpc:*/receive`
+-   `cl-protobufs.testing-rpc:*/close`
+-   `cl-protobufs.testing-rpc:*/cleanup`
+
+cells.
```

Comment:
Cells?  Not sure what that means. Also, I would use `<service-name>/start` etc, since `*/start` just looks like a literal Lisp symbol to me.